packages/storybook/config/storybook/jetPieThemeDocs.js

```diff
@@ -0,0 +1,8 @@
+import { create } from '@storybook/theming';
+
+export default create({
+    base: 'light',
+    colorPrimary: '#f36d00',
+    colorSecondary: '#4996FD',
```

Comment:
NIT: use consistent casing in the hex codes.

```suggestion
    colorSecondary: '#4996fd',
```

Reply:
Looks like this still needs actioning, @matthewhardern

Reply:
sorted